packages/common/src/content.ts

```diff
@@ -45,3 +45,57 @@ export function getCategory(itemType: string = ""): string {
 export function getTypes(category: string = ""): string[] {
   return categories[category.toLowerCase()];
 }
+
+/**
+ * ```js
+ * import { getType } from "@esri/hub-common";
+ * //
+ * getType(item)
```

Comment:
this example usage doesn't seem to match the signature below, seems like this should be `getType(layer)` which yields a string layer type, not `[ 'Hub Site Application' ]`.

Also a description of the intended use would help me understand this better - if `layer` is the first argument, I assume this is _primarily_ used to get the layer type.

Reply:
I was trying to accommodate what hub indexer currently does into hub.js. Actually for formatting AGO item results, we won't have a layer object, but we use the same function in hub indexer for layer and item slightly differently. How about I do just `getItemType(item)` which does not deal with layer?